jax/api.py

```diff
@@ -483,15 +501,18 @@ def jacfwd(fun: Callable, argnums: Union[int, Sequence[int]] = 0,
     A function with the same arguments as ``fun``, that evaluates the Jacobian of
     ``fun`` using forward-mode automatic differentiation.
 
+  >>> import jax
```

Comment:
Nit: I seem to remember that there is a way to set some common setup code, so that we don't have to repeat.

Reply:
I’ll look into that.

Reply:
Resolving as we can look at this in future.